pkg/search/postgres/query/time_query.go

```diff
@@ -62,6 +76,29 @@ func timeQueryEntry(ctx *queryAndFieldContext, prefix, formattedTime string) *Qu
 	}, nil)
 }
 
+func parseTimeRange(value string) (from, to time.Time, ok bool, err error) {
```

Comment:
nit:  A comment here or in the calling function calling out that it is expected query builder converted the times to milliseconds would have helped me when reviewing this.

Reply:
Yeah let me add a comment